SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute.iOS/Distribute.cs

```diff
@@ -10,14 +10,36 @@ public static partial class Distribute
         [Preserve]
         public static Type BindingType => typeof(iOSDistribute);
 
-        static bool PlatformEnabled { get; set; }
+        static bool PlatformEnabled
+        {
+            get
+            {
+                return iOSDistribute.IsEnabled();
+            }
 
+            set
+            {
+                iOSDistribute.SetEnabled(value);
+            }
+        }
         static void PlatformSetInstallUrl(string installUrl)
         {
+            iOSDistribute.SetInstallUrl(installUrl);
         }
 
         static void PlatformSetApiUrl(string apiUrl)
         {
+            iOSDistribute.SetApiUrl(apiUrl);
+        }
+
+        /// <summary>
+        /// Process URL request for the Distribute service.
+        /// Place this method call into app delegate openUrl method.
+        /// </summary>
+        /// <param name="url">The url with parameters.</param>
+        public static void PlatformOpenUrl(NSUrl url)
```

Comment:
Platform prefix for method name is only used because of the partial class pattern with portable. Since this is a platform specific API and public and documented, remove Platform prefix from this one.